### PR TITLE
Update the README structure

### PR DESCRIPTION
**Pull request by glonlas**
## Summary
Change the README structure to bring to the top the essential information for new visitors.
This new README is available here: https://github.com/freqtrade/freqtrade/blob/feature/revamp_readme/README.md

## Quick changelog

- Update README page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,33 +22,10 @@ expect.
...
-## Branches
-The project is currently setup in two main branches:
-- `develop` - This branch has often new features, but might also cause 
-breaking changes.
-- `master` - This branch contains the latest stable release. The bot 
-'should' be stable on this branch, and is generally well tested. 
+## Exchange marketplaces supported
+- [X] [Bittrex](https://bittrex.com/)
+- [X] [Binance](https://www.binance.com/)
+- [ ] [113 others to tests](https://github.com/ccxt/ccxt/). _(We cannot guarantee they will work)_
```
Comment (shusso) [inline README.md]: Good note 👍

File: README.md [hunk 2]
```diff
@@ -193,6 +142,46 @@ bot. More details on our
...
+### [Feature Requests](https://github.com/freqtrade/freqtrade/labels/enhancement)
+Have you a great idea to improve the bot you want to share? Please,
+first search if this feature was not [already discussed](https://github.com/freqtrade/freqtrade/labels/enhancement).
+If it hasn't been requested, please 
+[create a new request](https://github.com/freqtrade/freqtrade/issues/new) 
+and ensure you follow the template guide so that it does not get lost 
+in the bug reports.
+
+### [Pull Requests](https://github.com/freqtrade/freqtrade/pulls)
+Feel like our bot is missing a feature? We welcome your pull requests!
```
Comment (shusso) [inline README.md]: I think we should add a note that before starting to do any major feature work that open an issue where you describe what you are going to implement and/or talk to us on slack before.

What says @gcarq @vertti @xmatthias ?

Comment (gcarq) [reply README.md]: Sounds good to me

File: README.md [hunk 3]
```diff
@@ -22,33 +22,10 @@ expect.
...
-breaking changes.
-- `master` - This branch contains the latest stable release. The bot 
-'should' be stable on this branch, and is generally well tested. 
+## Exchange marketplaces supported
+- [X] [Bittrex](https://bittrex.com/)
+- [X] [Binance](https://www.binance.com/)
+- [ ] [113 others to tests](https://github.com/ccxt/ccxt/). _(We cannot guarantee they will work)_
 
 ## Features
 - [x] **Based on Python 3.6+**: For botting on any operating system -
```
Comment (xmatthias) [inline README.md]: "For trading on any operating system" ? (just a proposal ... )

File: README.md (hunk 2 above)
Comment (vertti) [reply README.md]: Yep.

**Pull request merged**